vagrant/ubuntu-xenial/Vagrantfile

```diff
@@ -9,9 +9,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.box = "ubuntu/xenial64"
   config.ssh.insert_key = false
-  config.vm.hostname = "demo.vagrant1.ala.org.au"
+  config.vm.hostname = "vagrant1.ala.org.au"
   config.vm.network :private_network, ip: "10.1.1.4"
   config.vm.network :forwarded_port, guest: 22, host: 2224, id: 'ssh'
+  config.vm.network :forwarded_port, guest: 5432, host: 5432 
+ #config.vm.network :forwarded_port, guest: 80, host: 8000
+  config.vm.synced_folder "/data/spatial-data", "/data/spatial-data"
```

Comment:
This folder sync isn't viable in this general vagrant file. If it is necessary for testing spatial, a new vagrant file should be created for it.

Reply:
You are correct. I have created a new Vagrant file. Exposing this port is for my dev box to connect into the database on vagrant